### DISCUSSION TITLE
Hy Linter

### DISCUSSION THREAD

**Discussion by jmwright** [hylang/hy#2508] [General]
Is there a lint package for Hy similar to black for Python? I have searched around (including on these Discussions and StackOverflow), but have not found anything that points to a linter.

**Comment by Kodiologist** [comment]
To my knowledge, no.

**Reply by Kodiologist** [reply]
To clarify, `black` is an autoformatter and doesn't produce any warnings like a linter does. But I don't know of any Hy autoformatters or Hy linters.

**Reply by jmwright** [reply]
Ok, thanks.

It's not directly relevant to this discussion, but for projects I work on we often run `black --diff --check .` to produce warnings without autoformatting.
